dataprofiler/tests/profilers/utils.py

```diff
@@ -11,8 +11,7 @@ def set_seed(seed=None):
     :param seed:
     :return:
     """
-    np.random.seed(seed)
-    random.seed(seed)
```

Comment:
Doesn't this still need to be set?

Reply:
ahh good catch, i do believe these would need to stay.